plugins/vpc-tunnel/config/netconfig.go

```diff
@@ -0,0 +1,221 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"net"
+	"strconv"
+	"strings"
+
+	"github.com/aws/amazon-vpc-cni-plugins/network/vpc"
+
+	log "github.com/cihub/seelog"
+	cniSkel "github.com/containernetworking/cni/pkg/skel"
+	cniTypes "github.com/containernetworking/cni/pkg/types"
+)
+
+// NetConfig defines the network configuration for the vpc-tunnel plugin.
+type NetConfig struct {
+	cniTypes.NetConf
+	DestinationIPAddress net.IP
+	VNI                  string
+	DestinationPort      uint16
+	Primary              bool
+	IPAddresses          []net.IPNet
+	GatewayIPAddress     net.IP
+	InterfaceType        string
+	Tap                  *TAPConfig
+}
+
+// netConfigJSON defines the network configuration JSON file format for the vpc-tunnel plugin.
+type netConfigJSON struct {
+	cniTypes.NetConf
+	DestinationIPAddress string   `json:"destinationIPAddress"`
+	VNI                  string   `json:"vni"`
+	DestinationPort      string   `json:"destinationPort"`
```

Comment:
Can you please swap lines 47 and 48, so that Dest IP address and port are in the correct order? Also applies to NetConfig struct above.